mimesniff/mime-type-sniffing/resources/generate_scriptable_mime_type_tests.py

```diff
@@ -0,0 +1,145 @@
+
+import collections
+import json
+
+#####
+
+BINARY_DATA_BYTES = ( b"\x00", b"\x01", b"\x02", b"\x03", b"\x04", b"\x05", b"\x06", b"\x07", b"\x08", b"\x0B", b"\x0E", b"\x0F", b"\x10", b"\x11", b"\x12", b"\x13", b"\x14", b"\x15", b"\x16", b"\x17", b"\x18", b"\x19", b"\x1A", b"\x1C", b"\x1D", b"\x1E", b"\x1F" )
+WHITESPACE_BYTES = ( b"\x09", b"\x0A", b"\x0C", b"\x0D", b"\x20" )
+TAG_TERMINATING_BYTES = ( b"\x20", b"\x3E" )
```

Comment:
While I think it's reasonable to expect readers to know \x20 is a space, I don't think it's reasonable to expect readers to know what 3E is, or why it's interesting here.  Add a comment?

Reply:
These concepts are taken directly from the spec.